edx_proctoring/templates/proctoring/seq_proctored_exam_submitted.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have submitted this proctored exam for review
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b> Pending </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      In general sessions are reviewed in 24-48 hours of submission. If you have questions about
```

Comment:
"Proctoring session results are usually available 24-48 hours after you submit your exam. If you have questions about the status of your session review after that time, contact edX Support."
(Link to https://www.edx.org/contact-us)